libs/blocks/chart/chart.js

```diff
@@ -282,7 +346,8 @@ export const getChartOptions = (chartType, data, colors, size) => {
 const initChart = (chartWrapper, chartType, data, colors, size) => {
   const themeName = getTheme(size);
   const chart = window.echarts?.init(chartWrapper, themeName, { renderer: 'svg' });
-  const chartOptions = getChartOptions(chartType, data, colors, size);
+  const chartOptions = getChartOptions(chartType, data, colors, size, chart);
```

Comment:
Seems like chart event listeners should be added separately, this is adding an unnecessary dependency to `getChartOptions`. 
Should the listeners be after `chart.setOption(chartOptions);`?

Reply:
I considered that, but it would require either processing the data, extracting the headers, etc twice or doing it all in initChart and passing a lot more arguments down through the functions. It didn't seem worth it. And anyway, the listeners are implementing changes to the donut series options, so it still made sense to me that they would be included in donutSeriesOptions.